tests/test_add_default_data.py

```diff
@@ -24,7 +24,7 @@ def test_add_default_data(qtbot, ab_app, monkeypatch):
     assert bw.projects.current == 'pytest_project'
 
     # The biosphere3 import finishes with a 'change_project' signal.
-    with qtbot.waitSignal(signals.change_project, timeout=10*60*1000):  # allow 10 mins for biosphere install
+    with qtbot.waitSignal(signals.database_changed, timeout=10*60*1000):  # allow 10 mins for biosphere install
```

Comment:
As we reload the project, I'm not sure it is wise to already continue with the tests before this is done. Tests may pass now, but could this cause unexpected behaviour in the future?